### DISCUSSION TITLE
when running a script in pycharm,haw to change default abaqus when i have installed both abaqus2021 and abaqus 2023?

### DISCUSSION THREAD

**Discussion by DengLuBing1** [haiiliin/abqpy#4798] [Q&A] [answered]
when i running a script in pycharm, the default abaqus is abaqus2023,i want to use abaqus 2021 to running this script,what should i do to change the default version of abaqus?

**Comment by haiiliin** [comment, marked as answer]
Set the environment variable `ABAQUS_BAT_PATH` to the file path of the `.bat` file, i.e.
```
C:\SIMULIA\Commands\abq2021.bat
```
Or simply set it to
```
abq2021
```
when the path `C:\SIMULIA\Commands` is added in the `Path` environment variable.

https://abqpy.readthedocs.io/en/stable/envvars.html#abaqus-bat-path

**Reply by DengLuBing1** [reply]
oh,last night i have set the right environment variable,but i forgot restart my computer,when i restart my computer,the problem fixed.
thank you for your help ! The abqpy package is very useful , i want to thank you for your package do me a big favour.
